packages/devextreme/testing/testcafe/tests/editors/textArea/index.ts

```diff
@@ -1,101 +1,101 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { testScreenshot } from '../../../helpers/themeUtils';
 import url from '../../../helpers/getPageUrl';
 import { createWidget } from '../../../helpers/createWidget';
-import TextArea from '../../../model/textArea';
-import { getPropertyValue, insertStylesheetRulesToPage } from '../../../helpers/domUtils';
-import { isMaterialBased } from '../../../helpers/themeUtils';
+import {
+  appendElementTo, setAttribute,
+} from '../../../helpers/domUtils';
+import { clearTestPage } from '../../../helpers/clearPage';
 
-const testFixture = () => {
-  if (isMaterialBased()) {
-    return fixture.disablePageReloads.skip;
-  }
-  return fixture.disablePageReloads;
-};
-
-testFixture()`TextArea_Height`
-  .page(url(__dirname, '../../container.html'));
+fixture.disablePageReloads`TextArea_Height`
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async () => clearTestPage());
 
 const text = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.';
+const longText = 'Prepare 2013 Marketing Plan: We need to double revenues in 2013 and our marketing strategy is going to be key here. R&D is improving existing products and creating new products so we can deliver great AV equipment to our customers.Robert, please make certain to create a PowerPoint presentation for the members of the executive team.';
```

Comment:
Why we need have 'longText'. Seems, that 'text' suits for us testing purpose.

Reply:
i agree sir let me remove it...